OSDOCS#3630: OTA etcd backup guidance (#71747)
openshift/openshift-docs: modules/update-etcd-backup.adoc

// Module included in the following assemblies:
//
// * updating/preparing_for_updates/updating-cluster-prepare.adoc

:_mod-docs-content-type: REFERENCE
[id="update-etcd-backup_{context}"]
= etcd backups before cluster updates

etcd backups record the state of your cluster and all of its resource objects.
You can use backups to attempt restoring the state of a cluster in disaster scenarios where you cannot recover a cluster in its currently dysfunctional state.

> In the context of updates, you can attempt an etcd restoration of the cluster in a case where an update introduced catastrophic conditions that cannot be fixed without reverting to the cluster version prior to the update.

Reviewer: ```suggestion
In the context of updates, you can attempt an etcd restoration of the cluster in a case where an update introduced catastrophic conditions that cannot be fixed without reverting to the cluster version before the update.
```
Vale flagged this.

> In the context of updates, you can attempt an etcd restoration of the cluster in a case where an update introduced catastrophic conditions that cannot be fixed without reverting to the cluster version prior to the update.

Reviewer: ```suggestion
In the context of updates, you can attempt an etcd restoration of the cluster in a case where an update introduced catastrophic conditions that cannot be fixed without reverting to the cluster version prior to the update.
```
I'm finding this sentence difficult to parse out. Maybe try to simplify it?

Author: How about this:
```suggestion
In the context of updates, you can attempt an etcd restoration of the cluster if an update introduced catastrophic conditions that cannot be fixed without reverting to the previous cluster version.
```

Reviewer: Yes, that's better!

etcd restorations might be destructive and destabilizing to a running cluster, use them only as a last resort.

[WARNING]
====
Due to their high consequences, etcd restorations are not intended to be used as a rollback solution.
Rolling your cluster back to a previous version is not supported.
If your update is failing to complete, contact Red{nbsp}Hat support.
====

> There are a number of factors that affect the viability of an etcd restoration.

Reviewer: ```suggestion
There are several factors that affect the viability of an etcd restoration.
```
Vale flagged this.
